<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opt</class>
 <widget class="QWidget" name="opt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>O P T(未来不再使用页面置换算法)</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>40</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
font-family:宋体;
font-size:30px;
background-color:rgb(160,160,160);
</string>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color:rgb(73,143,242,38%);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
font-family:宋体;
font-size:25px;
</string>
   </property>
   <property name="text">
    <string>结 果：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
font-family:宋体;
font-size:25px;
</string>
   </property>
   <property name="text">
    <string>请 输 入 访 问 序 号：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color:rgb(73,143,242,38%);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
font-family:宋体;
font-size:25px;
</string>
   </property>
   <property name="text">
    <string>请 输 入 块 数：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color:rgb(73,143,242,38%);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
font-family:宋体;
font-size:25px;
</string>
   </property>
   <property name="text">
    <string>请 输 入 页 面 数 量：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>861</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
background-color:rgb(73,143,242,38%);
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>650</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
font-family:宋体;
font-size:30px;
background-color:rgb(160,160,160);
</string>
   </property>
   <property name="text">
    <string>返 回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
